modules/utility/preview/glow/default.nix

```diff
@@ -0,0 +1,6 @@
+_: {
```

Comment:
again, `_:` can be ommitted in `default.nix` as it is used explicitly for imports.